packages/multiple-choice/configure/src/index.js

```diff
@@ -9,17 +9,37 @@ import ReactDOM from 'react-dom';
 import Root from './root';
 import debug from 'debug';
 import { choiceUtils as utils } from '@pie-lib/config-ui';
+import defaults from 'lodash/defaults';
 
 const log = debug('multiple-choice:configure');
 
+const prepareCustomizationObject = (model) => {
+  const defaultValues = {
+    promptLabel : 'Prompt',
+    responseTypeLabel: 'Response Type',
+    choicesLabel: 'Choice Labels',
+    addChoiceButtonLabel: 'Add a choice',
+    enableSelectChoiceLabel: true,
+    enableSelectResponseType: true,
+    enableAddChoice: true,
+    enableAddFeedBack: true,
+    enableDeleteChoice: true
+  };
+
+  return {
+    ...model,
+    config: defaults(model.config, defaultValues)
```

Comment:
can you use `model.configure` - 'configure' has a special meaning in the pie domain.

Reply:
Done @edeustace